rust/hyperlane-core/src/chain.rs

```diff
@@ -52,12 +52,14 @@ impl<'a> std::fmt::Display for ContractLocator<'a> {
 pub enum KnownHyperlaneDomain {
     Ancient8 = 888888888,
     Arbitrum = 42161,
+    Arthera = 10242,
```

Comment:
we can rm Arthera from this PR and just wait to apply the on-chain config to remove it

Reply:
Do we want to update the image tags in this PR now that we've removed references to Arthera?

Reply:
Nah that's okay. Their presence in the agents won't have any negative impact